frontend/src/styles/GlobalStyles.ts

```diff
@@ -33,16 +33,30 @@ export const GlobalStyles = createGlobalStyle`
     src: url('/fonts/Mont-Regular.otf') format('opentype');
   }
 
-  html {
-    scroll-behavior: smooth;
+  html, body {
```

Comment:
Why selector body here and again in 46line. Is not so common to put so many style in html.